.github/CODEOWNERS

```
@@ -195,3 +195,6 @@
 /modules/xresources.nix                               @rycee
 
 /modules/xsession.nix                                 @rycee
+
+/modules/programs/rofi-pass.nix                      @seylerius
+/tests/modules/programs/rofi-pass/*                  @seylerius
```

The `*` is unnecessary
```suggestion
/tests/modules/programs/rofi-pass                    @seylerius
```

Can also move the block to underneath `/modules/programs/powerline-go.nix` to maintain the alphabetic order.